docs/middleware.rst

```diff
@@ -458,6 +458,19 @@ this middleware to have any effect.
    >>> w3.middleware_onion.add(construct_sign_and_send_raw_middleware(acct))
    >>> w3.eth.default_account = acct.address
 
+The Infura Ethereum API only supports signed transactions. This often results in ``send_raw_transaction`` being used repeatedly, instead we can automate this process with ``construct_sign_and_send_raw_middleware(private_key_or_account)``.
```

Comment:
We're trying to keep our docs as API agnostic as possible. So instead of highlighting specifically the Infura API, maybe something like: 
```suggestion
:ref:`Hosted nodes<local_vs_hosted>` (like Infura or Alchemy) only support signed transactions. This often results in ``send_raw_transaction`` being used repeatedly. Instead, we can automate this process with ``construct_sign_and_send_raw_middleware(private_key_or_account)``.
```

Reply:
These edits are great, awesome work! :100: